pyproject.toml

```diff
@@ -11,6 +11,8 @@ python = "^3.12"
 fastapi = "^0.115.11"
 uvicorn = "^0.34.0"
 pydantic = "^2.10.6"
+pandas = "^2.1.4"
+sic-classification-library = {git = "https://github.com/ONSdigital/sic-classification-library.git"}
```

Comment:
Can you use a release version - v0.1.2 is the latest.

See sic-classification-utils repo pyproject.toml for an example.